cordova-lib/spec-plugman/platforms/ios.spec.js

```diff
@@ -301,14 +301,8 @@ describe('ios project handler', function() {
             });
         });
         it('of two plugins should apply xcode file changes from both', function(){
```

Comment:
@stevengill : if I modify this test, the console.logs both print out, but there are 3 failures.

``` it('of two plugins should apply xcode file changes from both', function(done){
             install('ios', temp, dummyplugin)
            .then(function (result) { 
                var xcode = ios.parseProjectFile(temp).xcode;
                // from org.test.plugins.dummyplugin
                expect(xcode.hasFile(slashJoin('Resources', 'DummyPlugin.bundle'))).toBeTruthy();
                expect(xcode.hasFile(slashJoin('Plugins','org.test.plugins.dummyplugin', 'DummyPluginCommand.h'))).toBeTruthy();
                expect(xcode.hasFile(slashJoin('Plugins','org.test.plugins.dummyplugin', 'DummyPluginCommand.m'))).toBeTruthy();
                expect(xcode.hasFile(slashJoin('Plugins','org.test.plugins.dummyplugin','targetDir','TargetDirTest.h'))).toBeTruthy();
                expect(xcode.hasFile(slashJoin('Plugins','org.test.plugins.dummyplugin','targetDir','TargetDirTest.m'))).toBeTruthy();
                expect(xcode.hasFile(slashJoin('SampleApp','Plugins','org.test.plugins.dummyplugin','Custom.framework'))).toBeTruthy();
                console.log('done 1')
            }).then(function (result) {
                install('ios', temp, weblessplugin)
            }).then(function (result) {
                var xcode = ios.parseProjectFile(temp).xcode;
                // from org.test.plugins.weblessplugin
                // Below fails and wants false
                expect(xcode.hasFile(slashJoin('Resources', 'WeblessPluginViewController.xib'))).toBeTruthy();
                // Below fails and wants false
                expect(xcode.hasFile(slashJoin('Plugins','org.test.plugins.weblessplugin','WeblessPluginCommand.h'))).toBeTruthy();
                // Below fails and wants false
                expect(xcode.hasFile(slashJoin('Plugins','org.test.plugins.weblessplugin','WeblessPluginCommand.m'))).toBeTruthy();
                expect(xcode.hasFile('usr/lib/libsqlite3.dylib')).toBeTruthy();
                console.log('done 2')
            });
            done();
        });
    }); ```